client/extensions/woocommerce/state/sites/payment-methods/test/actions.js

```diff
@@ -262,17 +271,49 @@ describe( 'actions', () => {
 								id: 'email',
 								label: 'PayPal email'
 							},
-							enabled: {
-								id: 'enabled',
-								label: 'Enabled',
-								type: 'checkbox',
-								value: 'no'
-							},
 						},
 						title: 'PayPal7',
 					}
 				} );
 			} );
 		} );
 	} );
+
+	describe( '#savePaymentMethodEnabled', () => {
+		const siteId = '789';
+
+		it( 'should dispatch an action', () => {
+			const getState = () => ( state );
+			const dispatch = spy();
+			savePaymentMethodEnabled( siteId, method )( dispatch, getState );
+			expect( dispatch ).to.have.been.calledWith(
+				{ type: WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE, siteId }
+			);
+		} );
+
+		it( 'should dispatch a success action with method complete', () => {
+			const getState = () => ( state );
+			const dispatch = spy();
+			const response = savePaymentMethodEnabled( siteId, method )( dispatch, getState );
+
+			return response.then( () => {
+				expect( dispatch ).to.have.been.calledWith( {
+					siteId: '789',
```

Comment:
Its a little confusing to see 789 here. I'd recommend just using the siteId shorthand instead like you did in the expect calls above and below